src/index.js

```diff
@@ -1,42 +1,77 @@
 import './style.css';
 
-const list = [
-  {
-    description: 'complete course',
-    index: 0,
-    complete: true,
-  },
-  {
-    description: 'Meet rina',
-    index: 1,
-    complete: true,
-  },
-  {
-    description: 'buy eggs',
-    index: 2,
-    complete: false,
-  },
-  {
-    description: 'go to gym',
-    index: 3,
-    complete: false,
-  },
-];
-const ul = document.querySelector('#myUL');
-// Create element
-const generate = (descValue) => {
-  const li = document.createElement('li');
-  const check = document.createElement('input');
-  const label = document.createElement('label');
-  check.type = 'checkbox';
-  ul.appendChild(li);
-  li.innerHTML = descValue;
-  li.appendChild(label);
-  label.appendChild(check);
-  check.setAttribute('class', 'check');
-  label.setAttribute('class', 'label');
-  li.setAttribute('class', 'items');
+const itemList = document.querySelector('#myUL');
+const tasksArray = JSON.parse(localStorage.getItem('todo')) || [];
+
+class Todo {
+  constructor(description) {
+    this.id = tasksArray.length + 1;
+    this.description = description;
+    this.completed = false;
+  }
+
+  static updateIndex = () => {
+    tasksArray.forEach((data, index) => {
+      data.id = index + 1;
+    });
+  };
+
+  static removeTask = (index) => {
+    tasksArray.splice(index - 1, 1);
+    this.updateIndex();
+  };
+}
+
+const storage = (todo) => {
+  todo.sort((a, b) => a.index - b.index);
+  localStorage.setItem('todo', JSON.stringify(todo));
 };
```

Comment:
- [ ] Your code is working fine but as per requirement, It should be a new JavaScript file for the new functionality. Kindly create a new js file and move your new functionality code there. :+1: